uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/LeaveRequestStatusMatrixTest.php

```diff
@@ -73,28 +73,101 @@ public function testCanTransitionToForLeaveApproverReturnsFalseForAllNonPossible
   }
 
   public function testCanTransitionToForAdminForReturnsTrueAllPossibleTransitionStatuses() {
+    $adminID = 5;
```

Comment:
After taking to @tunbola it seems the tests do not rely on existing data. These values are random and only tests that actually require a contact to be created will use the fabricator.